src/Xamarin.Android.Build.Tasks/Microsoft.NET.Workload.Android/WorkloadManifest.in.json

```diff
@@ -17,7 +17,7 @@
       "version": "@SDK_PACK_VERSION@",
       "alias-to": {
         "osx-x64": "Microsoft.Android.Sdk.osx-x64",
-        "win-x64": "Microsoft.Android.Sdk.win-x64"
+        "win-x64": "Microsoft.Android.Sdk.win-x64",
```

Comment:
This results in invalid JSON (JSON doesn't like trailing commas), which may or may not implicate "other" things.  For sanity, we should have only valid JSON.